pandas/tests/frame/methods/test_transpose.py

```diff
@@ -110,11 +110,18 @@ def test_transpose_float(self, float_frame):
             assert s.dtype == np.object_
 
     @td.skip_array_manager_invalid_test
-    def test_transpose_get_view(self, float_frame):
+    def test_transpose_get_view(self, request, float_frame, using_copy_on_write):
         dft = float_frame.T
-        dft.values[:, 5:10] = 5
-
-        assert (float_frame.values[5:10] == 5).all()
+        dft.iloc[:, 5:10] = 5
+
+        if using_copy_on_write:
+            # TODO(CoW) transpose
+            request.node.add_marker(
+                pytest.mark.xfail(reason="transpose doesn't yet do CoW")
+            )
+            assert (float_frame.values[5:10] != 5).all()
```

Comment:
transpose if doing CoW now, I think you can remove the xfail (CI is failing as well here)